lib/helper.ts

```diff
@@ -0,0 +1,17 @@
+export const filterFalsyValuesFromArray = <T>(
+  array: (T | undefined | null | false | 0 | "")[]
+): T[] => {
+  return array.filter(Boolean) as T[]
+}
+
+export const contentColorClasses = [
+  "bg-accents-pink",
+  "bg-accents-purple",
+  "bg-accents-orange",
+  "bg-accents-blue",
+] as const
+
+export const getRandomContentColorClass = () => {
+  const randomIndex = Math.floor(Math.random() * contentColorClasses.length)
+  return contentColorClasses[randomIndex]
+}
```

Comment:
Nice with global color functionality. Consider moving the functions to a file like: `lib/helpers/colors.ts`

Reply:
I created a global helper/ directory with different helper files - `helper.colors.ts` is one of them 😉